libraries/botbuilder-dialogs-adaptive/src/adaptiveDialog.ts

```diff
@@ -167,25 +185,36 @@ export class AdaptiveDialog<O extends object = {}> extends DialogContainer<O> {
     }
 
     public createChildContext(dc: DialogContext): DialogContext | undefined {
-        const state: AdaptiveDialogState<O> = dc.activeDialog.state;
+        const activeDialogState = dc.activeDialog.state;
+        let state = activeDialogState[this.adaptiveKey] as AdaptiveDialogState;
+
+        if (state == null) {
+            state = new AdaptiveDialogState();
```

Comment:
>AdaptiveDialogState [](start = 24, length = 19)

This should be an interface not a class as it ultimately gets persisted.  In JS/TS we use interfaces and plain old JSON objects for things that are persisted. That's because, unlike C#, class instances wont roundtrip in JS.